string old_name
string new_name
---
bool res
